pkgs/by-name/aw/aws-signing-helper/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "aws-signing-helper";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "aws";
```

Comment:
The `meta` section refers to https://github.com/aws/rolesanywhere-credential-helper, but this refers to https://github.com/aws/aws-signing-helper, which does not exist